frontend/javascripts/oxalis/view/action-bar/tracing_actions_view.tsx

```diff
@@ -684,6 +690,18 @@ class TracingActionsView extends React.PureComponent<Props, State> {
     }
 
     menuItems.push(screenshotMenuItem);
+
+    if (activeUser?.isSuperUser) {
```

Comment:
This is by design. Let's do a soft launch and try some larger datasets on the server first before releasing this publicly.